### PR TITLE
Exclude ui from Sonar and move navigation out of ui package

### PR DESCRIPTION
**Pull request by yzueger**
Actual time: 1h

### CODE REVIEW

**Review by KlaKalma — APPROVED** [review]
lgtm, thanks

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/DummyScreen.kt
Comment (joriba) [inline app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/DummyScreen.kt:1]: Do we still need this?

**Review by joriba — APPROVED** [review]
Do we still need the DummyScreen? 
Otherwise lgtm

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/DummyScreen.kt
Comment (yzueger) [reply app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/DummyScreen.kt:1]: No we don't, but I didn't want to delay the PR. I don't think deleting it later will cause merge conflicts

**Pull request merged**